Signal/test/util/HapticFeedback.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+protocol HapticAdapter {
```

Comment:
newline between imports and other file contents?

Reply:
Done.